topicgenerator/src/main/java/io/dockstore/topicgenerator/client/cli/TopicGeneratorClient.java

```diff
@@ -172,17 +170,17 @@ private void generateTopics(TopicGeneratorConfig topicGeneratorConfig, Iterable<
      * @param trsId
      * @param versionId
      * @param entryType
-     * @param descriptorContent
+     * @param descriptorFile
      */
-    private void getAiGeneratedTopicAndRecordToCsv(OpenAiService openAiService, CSVPrinter csvPrinter, String trsId, String versionId, String entryType, String descriptorContent) {
+    private void getAiGeneratedTopicAndRecordToCsv(OpenAiService openAiService, CSVPrinter csvPrinter, String trsId, String versionId, String entryType, FileWrapper descriptorFile) {
         // A character limit is specified but ChatGPT doesn't follow it strictly
         final String systemPrompt = "You will be provided with a " + entryType + ", and your task is to summarize it in one sentence where the first word is a verb. Use a maximum of 150 characters.";
```

Comment:
Maybe add that the first word should not be "summarize"

Reply:
Or at least not "Summarize the <entrytype>"

Reply:
I tried that and it doesn't always listen. I tweaked the prompt a little (not a meaningful change really), and it seemed to work